[DNS] Final review for go live (#3457)
cloudflare/cloudflare-docs: products/dns/src/content/dns-firewall/faq.md

<summary>How does DNS Firewall choose a backend nameserver to query upstream?</summary>
<div>

> DNS Firewall alternates between a customer's nameservers. Additionally, DNS Firewall uses an algoritm that more likely sends queries to the faster upstream nameservers and less likely to the slower nameservers.

Reviewer: ```suggestion
DNS Firewall alternates between a customer's nameservers, using an algorithm is more likely to send queries to the faster upstream nameservers than slower nameservers.
```

EDNS limits the effectiveness of the DNS cache.

</Aside>

> Some resolvers might not be sending any EDNS data. By setting the `ecs_fallback` parameter to `true` via the [API](https://api.cloudflare.com/#dns-firewall-update-dns-firewall-cluster) DNS Firewall will forward the IP subnet of the resolver instead only if there is no EDNS data present in incoming the DNS query.

Reviewer: ```suggestion
Some resolvers might not be sending any EDNS data. When you set the `ecs_fallback` parameter to `true` via the [API](https://api.cloudflare.com/#dns-firewall-update-dns-firewall-cluster), DNS Firewall will forward the IP subnet of the resolver instead only if there is no EDNS data present in incoming the DNS query.
```

</div>
</details>

<details>
<summary>Does DNS Firewall cache negative answers?</summary>
<div>

Not by default, but you can set `negative_cache_ttl` via the [API](https://api.cloudflare.com/#dns-firewall-update-dns-firewall-cluster). This will affect the TTL of responses with status `REFUSED` or `NXDOMAIN`. 

</div>
</details>

<details>
<summary>Does DNS Firewall cache responses with status SERVFAIL?</summary>
<div>

No. If the upstream nameserver responds with a `SERVFAIL`, DNS Firewall will try again on the next query and not cache that response.
  
</div>
</details>

<details>

> <summary>How to set PTR records for nameserver hostnames?</summary>

Reviewer: ```suggestion
<summary>How can I set PTR records for nameserver hostnames?</summary>
```

<div>

> If you want PTR records on the assigned DNS Firewall cluster IPs that point to your nameserver hostnames, please reach out to your Cloudflare account team or Cloudflare support.

Reviewer: ```suggestion
If you want PTR records on the assigned DNS Firewall cluster IPs that point to your nameserver hostnames, please reach out to your Cloudflare account team.
```
